specification/threatmodel.adoc

```diff
@@ -1,187 +1,296 @@
 data e.g. via shared caches, branch predictor poisoning, page-faults.
 
-T15: Leveraging architectural side-channel attacks due to shared cache and 
+T15: Leveraging architectural side-channel attacks due to shared cache and
 other shared resources e.g. via prime/probe, flush/reload approaches
 
-T16: Malicious access to ciphertext with known plaintext to launch a 
-dictionary attack on TVMs or TSM or trusted firmware to extract 
+T16: Malicious access to ciphertext with known plaintext to launch a
+dictionary attack on TVMs or TSM or trusted firmware to extract
 confidential data.
 
-T17: Tamper of TVM state during migration of a TEE workload assets within 
-the platform or from one platform to another. 
+T17: Tamper of TVM state during migration of a TEE workload assets within
+the platform or from one platform to another.
 
 T18: Forging of attestation evidence and sealed data associated with a TVM.
 
-T19: Stale TLB translations (for U/HS mode or for VU/VS) created during TSM 
-or TVM operations are used to execute malicious code in the TVM (or consume 
+T19: Stale TLB translations (for U/HS mode or for VU/VS) created during TSM
+or TVM operations are used to execute malicious code in the TVM (or consume
 stale/invalid data)
 
-T20: Isolation of performance monitoring and/or debug state for a TVM 
-leading to information loss via performance monitoring events/counters and 
+T20: Isolation of performance monitoring and/or debug state for a TVM
+leading to information loss via performance monitoring events/counters and
 debug mode accessible information.
 
 T21: A TVM causes a denial of service on the platform
 
 [NOTE]
 ====
-This is not an exhaustive list and will be updated on a regular basis as attacks evolve._
+This threat model is not an exhaustive list and will be updated (via the RISC-V
+Security Model specification <<R21>>) on a regular basis as attacks evolve.
 ====
 
 === Scope
 
-This specification does not prescribe the scope of mitigation and focusses 
-on the TEEI interface and use-of/impact-on the RISC-V ISA. It is 
-recommended that implementations of this reference architecture address 
-threats from system software adversaries. Implementations may choose to 
-mitigate threats from additional adversaries. For all cases, denial of 
-service by TVMs must be prevented. At the same time, denial of service by 
+This specification does not prescribe the scope of mitigation and focusses
+on the TEEI interface and use-of/impact-on the RISC-V ISA. It is
+recommended that implementations of this reference architecture address
+threats from system software adversaries. Implementations may choose to
+mitigate threats from additional adversaries. For all cases, denial of
+service by TVMs must be prevented. At the same time, denial of service by
 host software is considered out of scope.
 
 [[design_survey]]
-=== TVM Security Requirements to address threat model 
+=== TVM Security Requirements to address threat model
 
```

Comment:
All these are just space removal changes until this point ?

Reply:
yes - no major changes in this section :)